### DISCUSSION TITLE
How to use A-B repeat feature

### DISCUSSION THREAD

**Discussion by 2412khoidz** [clappr/clappr#2183] [Q&A] [answered]
This feature is great and virtually every mainstream media player has it. Does clapper have A-B repeat in its GUI? If yes, how can I use it. Thanks

**Comment by leaofelipe** [comment, marked as answer]
Hi! @2412khoidz!
Natively, we don't have that feature in Clappr. However, the main characteristic of our project is the extensive customization possibilities and the creation of new plugins.

It would be great to have an A-B Repeat plugin made by the community! 🎉🎉🎉
